src/ostorlab/cli/scan/run/run.py

```diff
@@ -68,6 +67,13 @@
     multiple=True,
     default=[],
 )
+@click.option(
+    "--no-follow",
```

Comment:
```suggestion
    "--unfollow",
```
--unfollow is more intuitive for me.